CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- Added feature: selectively ignore some of the measures (#416)

```

Comment:
```suggestion
Add: explicitAttrs flag (in configuration and also group/device provisioning) to progress only the measures corresponding to attributes declared in attributes array (#416)
```

to align with a similar text in the associated PR in the ioagent-node-lib

Reply:
Fixed in 83834fe